src/App.jsx

```diff
@@ -0,0 +1,88 @@
+import { useEffect, useState } from "react";
+import TodoInput from "./components/TodoInput/TodoInput";
+import TodoList from "./components/TodoList/TodoList";
+import Card from "./components/ui/Card";
+
+function App() {
+	const [inputText, setInputText] = useState("");
+	const [todosList, setTodosList] = useState([]);
+
+	// ! Updating name of task when getting edited
+	const updateTodoName = (updatedValue, id) => {
+		const newTasks = todosList.map(todo => (todo.id === id ? { ...todo, name: updatedValue } : todo));
+		setTodosList([...newTasks]);
+	};
+
+	// ! Getting previous todo list from LocalStorage
+	useEffect(() => {
+		if (JSON.parse(localStorage.getItem("storedTasks")))
+			setTodosList([...JSON.parse(localStorage.getItem("storedTasks"))]);
+	}, []);
+
+	// ! Updating current todo list to LocalStorage
+	useEffect(() => {
+		setTimeout(() => {
```

Comment:
Po co ten setTimeout?

Reply:
Bez timeout'a zaciągało mi do LocalStorage pusty array na początku i zerowało obecnie zapamiętany. 
Pewnie można było to jakoś obejść, ale zostawiłem to sobie na później.